src/tests/ftest/util/run_utils.py

```diff
@@ -474,6 +474,12 @@ def run_remote(log, hosts, command, verbose=True, timeout=120, task_debug=False,
     task.set_info('fanout', fanout)
     # Enable forwarding of the ssh authentication agent connection
     task.set_info("ssh_options", "-oForwardAgent=yes")
+
+    # Support running remote commands with the test environment
+    _test_env_file = os.environ.get("DAOS_TEST_ENV_FILE", None)
+    if _test_env_file is not None:
+        command = f"source {_test_env_file} && {re.sub(r'\bsudo\b', 'sudo -E', command)}"
```

Comment:
If we use `-E` here maybe we should also use it for all `sudo` everywhere. Otherwise local commands will not run with the same env as remote